test/OWNERS

```diff
@@ -10,20 +10,17 @@ reviewers:
   - lavalamp
   - liggitt
   - neolit123
+  - pohly
   - SataQiu
   - smarterclayton
   - thockin
   - wojtek-t
 approvers:
   - andrewsykim
-  - aojea
-  - BenTheElder
   - bowei # for test/e2e/{dns*,network}.go
   - caseydavenport # for test/e2e/{dns*,network}.go
-  - cblecker
   - dchen1107
   - deads2k
-  - dims
```

Comment:
Am i losing my privileges @pohly ?

Reply:
Oops, yes, you are.

I thought you were in sig-testing-approvers, but you aren't. That was only sig-testing-reviewers.

Fixed.

Reply:
> Oops, yes, you are.

thanks @pohly !